scripts/benchmark.js

```diff
@@ -18,38 +18,38 @@ const packagename = process.argv[2];
 var filename = process.argv[3];
```

Comment:
I'd squash this change into the last one so they never see your treachery :see_no_evil: